### PR TITLE
phan: add page

### PR DESCRIPTION
**Pull request by owenvoke**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/phan.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Generate a Phan configuration file using a specific level (1 being strictest to 5 being the least strict):
+
+`phan --init --init-level {{level}}`
+
+- Analyse the current directory:
+
+`phan`
+
+- Analyse one or more directories:
```
Comment (navarroaxel) [inline pages/common/phan.md:18]: space separated list of directories?

Comment (owenvoke) [reply pages/common/phan.md:18]: Looks like you have to chain the `--directory` flag. So:
`phan --directory {{path/to/directory1}} --directory {{path/to/directory2}}`

File: pages/common/phan.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`phan --directory {{path/to/directory}}`
```
Comment (sbrl) [inline pages/common/phan.md:20]: What about this?
```suggestion
`phan --directory {{path/to/directory}} --directory {{path/to/another_directory}}`
```

**Review by sbrl — APPROVED** [review]
Looks ok to me! I've left a minor comment, but it's optional :-)

**Review by Rudloff — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**